src/components/comment-list.tsx

```diff
@@ -20,7 +19,7 @@ export const CommentList: React.VoidFunctionComponent = () => {
   return (
     <div>
       <CommentsHeader>Комментарии - {comments.length}</CommentsHeader>
-      {comments.map((comment: Comment) => {
+      {comments.map((comment) => {
```

Comment:
`: Comment` is a noop because `comments` are of type `PublicCommentInfo[]`. We have the right typing in `map` automatically.